.github/workflows/lint-test.yaml

```
@@ -129,9 +108,14 @@ jobs:
         uses: helm/kind-action@v1.10.0
         if: steps.list-changed.outputs.changed == 'true'
 
-      - name: Run chart-testing (install)
+      - name: Run chart-testing (install ${{ matrix.test_cases.name }})
         id: install
         if: steps.list-changed.outputs.changed == 'true'
-        run: |
-            ct install --target-branch ${{ github.event.repository.default_branch }} \
-            --helm-extra-set-args "--set=postgresql.enabled=true --set=postgresql.global.postgresql.auth.password=testing123456 --set=internalDatabase.enabled=false --set=externalDatabase.enabled=True --set=externalDatabase.type=postgresql --set=externalDatabase.password=testing123456"
+        run: ct install --target-branch ${{ github.event.repository.default_branch }} ${{ matrix.test_cases.helm_args }}
+
+  summary:
+    runs-on: ubuntu-latest-low
+    needs: [changes, run-tests]
```

```suggestion
    needs: [changes, run-tests]
    if: always()
```
https://github.com/nextcloud/.github/blob/e8ef9164890fe4bf40bf4132dfa6656834a40e25/workflow-templates/node-test.yml#L104